docs/architecture.md

```diff
@@ -0,0 +1,52 @@
+# Architecture
+
+Newman Dashboard is a real-time reporter for newman to control, view and debug runs on a WebUI dashboard.
+
+## Pub - Sub
```

Comment:
Does not make a lot of sense to name this section Pub - Sub. Can we think of  a better name? 🤔


Suggestions: `Core concepts` or `Basic model`